.gitignore

```diff
@@ -25,6 +25,8 @@ HELP.md
 .sts4-cache
 
 ### IntelliJ IDEA ###
+/frontend/.idea/*
+/backend/.idea/*
```

Comment:
I opened the project as two different projects in IntelliJ and had to add these .idea files to the .gitignore.

Reply:
Line 30 already ignores it right?

Reply:
It doesn't seem to work that way.